### PR TITLE
fix(identifier): emoji hash line height

### LINKED ISSUE #17330: Onboarding identifier carousel spacing
# Bug Report

The spacing/height of the identifier carousel card in the onboarding flow is not correct, design comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=5689-66197&mode=design)
## Problem 
[comment]: # (A problem description)

The height/spacing should match design, bug reported here  https://github.com/status-im/status-mobile/pull/17293#pullrequestreview-1630620781

### Additional Information

- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by BalogunofAfrica**
Fixes https://github.com/status-im/status-mobile/issues/17330, follow up to https://github.com/status-im/status-mobile/pull/17293

### Summary
The identifier card height in the onboarding flow should match the design

#### Platforms

- Android
- iOS

status: ready

### CODE REVIEW

File: src/status_im2/contexts/onboarding/identifiers/profile_card/style.cljs [hunk 1]
```diff
@@ -66,4 +66,5 @@
           {:margin-top 2})))
 
 (def emoji-hash
-  {:margin-top 12})
+  {:margin-top  12
+   :line-height 20.5})
```
Comment (flexsurfer) [inline src/status_im2/contexts/onboarding/identifiers/profile_card/style.cljs:70]: are you sure about `.5` ?

Comment (BalogunofAfrica) [reply src/status_im2/contexts/onboarding/identifiers/profile_card/style.cljs:70]: Yeah, the lineHeight of `20.5` matches the designs' spacing and height. 

The design uses regular images in a row for the emoji hash while we use text for this, and the lineHeight of the text used initially was `21.75` (`paragraph-1` in typography).

**Review by flexsurfer — APPROVED** [review]

**Review by smohamedjavid — APPROVED** [review]

**Comment by churik** [comment]
Best to send it to design review, as it is follow-up on design issue.

e2e failures rare not related.

**Review by Francesca-G — COMMENTED** [review]
Card height still doesn't match:

Screenshot 2023-09-25 alle 10 45 57

I think the spacing between the elements in the card is different than design and that may affect card height.

**Comment by BalogunofAfrica** [comment]
@Francesca-G 🤔

It seemed to match when I overlayed implementation against the design

Screenshot 2023-09-25 at 10 13 36

**Comment by Francesca-G** [comment]
> @Francesca-G 🤔
> 
> It seemed to match when I overlayed implementation against the design
> 
> Screenshot 2023-09-25 at 10 13 36

I did the same but this was the result on [Figma](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=1902%3A125364&mode=design&t=hsLLSGxFsH38BCS2-1)
Please let me know if it's a problem on my side :)

**Comment by BalogunofAfrica** [comment]
@Francesca-G Can you please retry 🙂. I just confirmed again and it matches the height.

**Comment by Francesca-G** [comment]
> @Francesca-G Can you please retry 🙂. I just confirmed again and it matches the height.
> 
it doesn't match on my side 🤔 I don't know if it's a screen res problem, I'm using an iPhone 13 to test this.

As you can see all the alignments inside the card don't match the design as well, so I think that's affecting the card height (which should be 184px)

Screenshot 2023-09-28 alle 14 09 54

**Comment by siddarthkay** [comment]
Hi @Francesca-G 
I helped @BalogunofAfrica test this by using iPhone 11 Pro as reference.
I would take a screenshot from simulator then paste it in figma and resize width to 375 and then compare.
and cc @J-Son89 I think we were told by Design team to use that as a reference.

**Comment by J-Son89** [comment]
> Hi @Francesca-G I helped @BalogunofAfrica test this by using iPhone 11 Pro as reference. I would take a screenshot from simulator then paste it in figma and resize width to 375 and then compare. and cc @J-Son89 I think we were told by Design team to use that as a reference.

yes you need to be using iphone 11 Pro to verify with Figma in this way, otherwise the sizes won't work. Additionally the screenshot needs to be from the phone/simulator (e.g not the laptop screenshot) so that the size is correct. 
Afaik the iphone 13 won't be accurate here 👍

**Comment by siddarthkay** [comment]
Just realised we have this documented here : https://github.com/status-im/status-mobile/blob/develop/doc/pixel-perfection.md

**Comment by BalogunofAfrica** [comment]
Hi @Francesca-G 🙂. Can this be merged?

**Comment by Francesca-G** [comment]
> Hi @Francesca-G 🙂. Can this be merged?

Yes, apologies again for the device resolution problem we encountered.
Moving it to the Merge column 🚀

**Pull request merged**